package ExceptionHandlingRules;

/*public class Rule_2 {
    public static void main(String[] args) {
        //You can't use multiple try block & single catch block

        int a[] = {1, 2, 3, 4, 5};
        try {
            System.out.println(a[6]);
        }
        try {
            System.out.println(a[6]);
        }
        catch (ArrayIndexOutOfBoundsException ai) {
            ai.printStackTrace();
        }
        System.out.println("Good Evening");
    }
}*/
